R/boxr__internal_verb_exit.R

```diff
@@ -49,17 +49,38 @@ returnDwOp <- function(op_detail) {
       "new local directories created",
       "new remote directories created"
     )
-  
+  # print("op")
+  # print(op_detail$files)
   file_list <- 
     lapply(
       items_list,
       function(item)
         suppressWarnings(
-          data.frame(dplyr::bind_rows(lapply(
-            op_detail$files, function(x) data.frame(x[item])
-          )))
+          
+          lapply(op_detail$files,
```

Comment:
I think this was the thing fouling up the tests (I suspect you came to the same conclusion).

My solution looks like:

```r
  file_list <- 
    purrr::map(
      items_list,
      function(item) {
        purrr::map_dfr(op_detail$files, ~.x[item])
      }
    )
```

But I have no idea if it is doing what Brandon intended...

Reply:
I think this is essentially a function to stack the JSON elements from an API response into a data frame.

I found the error to be in `bind_rows()` (vectors related lower) when it wouldn't stack `octmode` and `character` values. I know `map_dfr()` calls `bind_rows()` so I guess we'd hit the same error.

The `as.character()` hack is yucky I agree. But since the Box API seems to take all of its inputs as strings. I figured the usability impact would be negligible.

Thanks for the review notes, I'll start responding tonight in bulk.